src/browsers/devicefarm.ts

```diff
@@ -13,34 +10,17 @@ export interface DevicefarmOptions {
   retryCount?: number;
   projectArn: string;
   expiresInSeconds?: number;
-  credentials: CredentialsOptions;
```

Comment:
Even if the type says it is required, it is not used anywhere. Removed as unused code